build/Linux+BSD/portable/copy-libs

```diff
@@ -116,8 +239,6 @@ building_on_CentOS_6() {
   copyLib libQt5Concurrent.so.5
   copyLib libQt5DBus.so.5
   copyLib libQt5Designer.so.5
-  copyLib libQt5Location.so.5
-  copyLib libQt5QuickParticles.so.5
```

Comment:
I removed these, and have tried out x86-64 app image without it https://bintray.com/ericfont/MuseScoreDevelopment/MuseScoreDev-armcross-x86_64/201603292308-armcross-914280c/view#files and it seems to work fine.  I enabled and tried all the plugins, and they seem to all work.